spec/linkedList.js

```diff
@@ -0,0 +1,175 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import LinkedList from '../src/linkedList'
+
+chai.use(chaiChange)
+
+before( () => {
+  let linkedList = new LinkedList()
+  linkedList.insert('apple')
+  linkedList.insert('banana')
+})
```

Comment:
It seems like you want this function to be run before each test.  You could use `beforeEach` for this (`before` only runs once).

Also, the way `linkedList` is declared on line 8 means that this variable is only defined within the block on lines 7 through 11.

But all the tests need access to `linkedList` so this must be fixed.

One way to fix this is to declare `linkedList` on line 6 (global scope) without assigning it a value and then assign a value to `linkedList` within the beforeEach like you are already doing.

So, line 6 would become `let linkedList` and line 8 would be `linkedList = new LinkedList()`.

Then you could remove these three lines in each of the tests,
```
const linkedList = new LinkedList()
linkedList.insert('apple')
linkedList.insert('banana')
```